core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -281,17 +242,6 @@ public synchronized void awaitLosePrimacy() throws IOException {
         "Should already be set to SECONDARY state. unexpected state: " + mState.get());
     Preconditions.checkState(mWriter != null, "writer thread must not be null in primary mode");
     Preconditions.checkState(mTailerThread == null, "tailer thread must be null in primary mode");
-<<<<<<< HEAD
-||||||| parent of b358b1a6a3... Prevent secondary UFS journal from modifying journal files
-    // Close async writer first to flush pending entries.
-    mAsyncWriter.close();
-    mAsyncWriter = null;
-=======
-
-    // Close async writer first to flush pending entries.
-    mAsyncWriter.close();
-    mAsyncWriter = null;
->>>>>>> b358b1a6a3... Prevent secondary UFS journal from modifying journal files
     mWriter.close();
     mWriter = null;
     mAsyncWriter = null;
```

Comment:
Is `mAsyncWrite.close()` missing ?

Reply:
It is missing because in 1.8, it never called `mAsyncWriter.close()`. In 1.8, `mAsyncWriter` doesn't even have a `close()` method.